docs/contact_us/contact.md

```diff
@@ -0,0 +1,11 @@
+# Contact Us
+
+Please feel free to get in touch with us using the following contact information:
+
+## Contact Information
+
+- **Email**: [contact@example.com](mailto:contact@example.com)
```

Comment:
I think The contact information should map to the github discussion or the gihub issues